src/multichainhelpers.ts

```diff
@@ -110,22 +117,22 @@ export async function refillFirstChain(signer: MultiChainSigner): Promise<void>
         sender: holderIdentity,
         recipient: refillDistibutor.address,
         tokenTicker: token,
-        wholeAmount: refillAmount(token),
+        amount: refillAmount(token),
       });
     }
   }
-
   if (jobs.length > 0) {
     for (const job of jobs) {
       logSendJob(signer, job);
-      await sendOnFirstChain(signer, job);
+      await sendOnFirstChain(profile, signer, job);
+      await sleep(waitFaucetTx);
```

Comment:
use `await sleep(50);` fixed to avoid the index update issue that we found in the wallet